app/assets/stylesheets/application.scss

```diff
@@ -77,6 +77,10 @@ body {
   font-weight: bold;
 }
 
+.text--underline {
+  text-decoration: underline;
+}
+
```

Comment:
I see that we have other typographic styles in this file, but ideally, style classes like `.text--underline` should be in `atoms/typography` if possible. Non-blocking, but would be a small step to better CSS organization.

Reply:
ah yeah just saw the bold one here already..but will move all the typographic stuff to the right place, thanks!